src/components/PivotTable/Draggable/Draggable.tsx

```diff
@@ -29,7 +34,7 @@ export interface DraggableProps<T> {
    * Called when an arrow key is pressed
    * You may want when the user presses an arrow key to move the draggable from one dropable to another
    */
-  onKeyNav: (dir: 'left' | 'right' | 'up' | 'down' | null, origin: string, key?: keyof T) => void
+  onKeyNav?: (dir: 'left' | 'right' | 'up' | 'down' | null, origin: string, key?: keyof T) => void
```

Comment:
Its possible create a type for this function signature and use here and in `Droppable.tsx` prop?